t/test.py

```diff
@@ -408,6 +408,34 @@ def test_strptime_handles_alphabets_in_format(self):
 
         self.assertEqual(dt1, dt2)
 
+    def test_strptime_special_chars(self):
+        date_string = "[1363*6*6] ? (12+13+14)"
+        date_format = "[%Y*%m*%d] ? (%H+%M+%S)"
+        dt1 = jdatetime.datetime.strptime(date_string, date_format)
+        dt2 = jdatetime.datetime(1363, 6, 6, 12, 13, 14)
+
+        self.assertEqual(dt1, dt2)
+
+    def test_strptime_small_y(self):
+        self.assertEqual(
+            jdatetime.datetime(1468, 1, 1),
+            jdatetime.datetime.strptime("68/1/1", "%y/%m/%d"))
+        self.assertEqual(
+            jdatetime.datetime(1369, 1, 1),
+            jdatetime.datetime.strptime("69/1/1", "%y/%m/%d"))
+
+    def test_strptime_do_not_match_excessive_characters(self):
+        self.assertRaises(
+            ValueError, jdatetime.datetime.strptime, '21 ', '%y')
```

Comment:
please change it to context manager style https://github.com/slashmili/python-jalali/blob/5f737a55fde5aa844c83769deb2af4a1b08e755b/t/test.py#L158

Reply:
Done.